README.md

```diff
@@ -95,9 +81,9 @@ android {
 
 ### Using AWS Mobile Client
 
-Authentication in Amplify for Android is dependent on AWS Mobile Client. Please refer to the [this documentation](https://aws-amplify.github.io/docs/android/authentication) for details.
+Amplify relies on the AWS Mobile Client for authentication with AWS services. Please refer to the [this documentation](https://aws-amplify.github.io/docs/android/authentication) for details.
 
-You need to ensure `AWSMobileClient` is [initialized](https://aws-amplify.github.io/docs/android/authentication#initialization) before you initalize and configure `Amplify`. Please note that AWSMobileClient initialization is not required when using API or Datastore category with API Key as the authorization mode.
+You need to ensure `AWSMobileClient` is [initialized](https://aws-amplify.github.io/docs/android/authentication#initialization) before you initalize and configure `Amplify`. Please note that AWSMobileClient initialization is not required when using API or Datastore category with _API Key_ as the authorization mode.
```

Comment:
https://docs.amplify.aws/lib/auth/getting-started?platform=android#initialization

It no longer goes to `initialization` section of the doc due to redirect

Reply:
updated